tardis/adapters/batchsystems/htcondor.py

```diff
@@ -17,7 +17,7 @@
 
 
 async def htcondor_status_updater(
-    options: AttributeDict, attributes: AttributeDict
+    options: AttributeDict, attributes: AttributeDict, executor=ShellExecutor
```

Comment:
I wanted a fallback executor in case none is provided (which should not happen, so it is pointless) but apparently I forgot the brackets it instantiate the class. `executor: Executor` makes the most sense.